frontend/src/lib/lemon-ui/LemonTabs/LemonTabs.scss

```diff
@@ -48,19 +52,24 @@
     .LemonTabs--transitioning & {
         transition: color 200ms ease;
     }
+
     &:not(:last-child) {
         margin-right: 2rem;
     }
+
     &:hover {
         color: var(--link);
     }
+
     &:active {
         color: var(--primary-dark);
     }
+
     &.LemonTabs__tab--active {
         color: var(--link);
-        text-shadow: 0 0 0.25px currentColor; // Simulate increased weight without affecting width
+        text-shadow: 0 0 0.25px currentcolor; // Simulate increased weight without affecting width
```

Comment:
Oh, apparently both work and which one is better is [contentious](https://github.com/stylelint/stylelint/issues/5863) because the upcoming [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/) says lowercase, but the currently accepted [CSS Color Module Level 3](https://www.w3.org/TR/css-color-3/) says camelCase. Well, let's use the spec that's _currently_ in force. ;)